### PR TITLE
Format code in code block

### PR DESCRIPTION
**Pull request by prajwalbajracharya**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

File: guide/english/css/class-selector/index.md [hunk 1]
```diff
@@ -7,11 +7,12 @@ A Class Selector is used in a CSS file to apply style to the HTML elements with
 To select elements with a specific class, we use a (.) named as period character, with the name of the class.
 
 For example
+```
 .center {
     text-align: center;
```
Comment (thecodingaviator) [inline guide/english/css/class-selector/index.md]:
```suggestion
  text-align: center;
```

File: guide/english/css/class-selector/index.md [hunk 2]
```diff
@@ -7,11 +7,12 @@ A Class Selector is used in a CSS file to apply style to the HTML elements with
...
     color: red;
```
Comment (thecodingaviator) [inline guide/english/css/class-selector/index.md]:
```suggestion
  color: red;
```

**Comment by thecodingaviator** [comment]
We use only two spaces for indentation, please make the changes

**Comment by RandellDawson** [comment]
@thecodingaviator This one has gone stale.

**Comment by thecodingaviator** [comment]
Hi @bajracharya1994 

As there has been no response from your end for a while, we are closing this PR.

Feel free to request this PR to be reopened, if you still wish to make the changes you originally added.  Also, remember to update your fork, so you have the latest version of master in case it has changed significantly.

Happy Coding! ✈️

**Pull request closed without merging**